app/views/pages/dashboard.html.erb

```diff
@@ -3,13 +3,27 @@
     <div>
       <h1 class="sr-only">Dashboard</h1>
       <p class="text-xl font-medium text-gray-900 mb-1"><%= t(".greeting", name: Current.user.first_name ) %></p>
-      <p class="text-gray-500 text-sm"><%= t(".subtitle") %></p>
+      <% if !@accounts.blank? %>
+        <p class="text-gray-500 text-sm"><%= t(".subtitle") %></p>
+      <% end %>
     </div>
-    <%= link_to new_account_path, class: "flex text-white text-sm font-medium items-center gap-1 bg-gray-900 rounded-lg p-2", data: { turbo_frame: "modal" } do %>
+    <%= link_to new_account_path, class: "flex text-white text-sm font-medium items-center gap-1 bg-gray-900 rounded-lg p-2 pr-3", data: { turbo_frame: "modal" } do %>
       <%= lucide_icon("plus", class: "w-5 h-5") %>
       <span><%= t(".new") %></span>
     <% end %>
   </header>
+  <% if @accounts.blank? %>
```

Comment:
In order to use the `@accounts` instance variable, it will need to be defined within the `pages_controller#dashboard` action.

Since we already have an `@account_groups` variable defined there, I'd recommend a small refactor like so:  

```ruby
@accounts = Current.family.accounts
@account_groups = @accounts.by_group(period: @period, currency: Current.family.currency)
```